certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+				"operationId": "GetInheritLinks",
+				"security": [
+					{
+						"oauth2_auth": []
+					}
+				],
+				"parameters": [
+					{
+						"name": "listName-dynamic",
+						"type": "string",
+						"in": "path",
+						"description": "Select the Table you want to get items from.",
+						"required": true,
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Select Source Table",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					},
+					{
+						"in": "path",
+						"name": "itemId",
+						"required": true,
+						"description": "Numeric ID of the item to get",
+						"type": "integer",
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Item Id"
+					},
+					{
+						"name": "type",
+						"type": "string",
+						"in": "query",
+						"description": "Select the Table you want to get items for.",
+						"required": true,
+						"x-ms-summary": "Select Target Table",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "XXX SOMETHING",
+						"schema": {
+							"type": "array",
+							"items": {
+								"type": "string"
+							}
+						}
+					}
+				}
+			}
+		},
+		"/lists/{listName-dynamic}/items/{itemId}/attachments": {
+			"post": {
+				"summary": "Add an attachment record to an item in your RAPID site",
+				"description": "Adds an attachment record to an item in your RAPID site. This record refers to the location of a file stored in SharePoint.",
+				"operationId": "SetAttachments",
+				"security": [
+					{
+						"oauth2_auth": []
+					}
+				],
+				"parameters": [
+					{
+						"name": "listName-dynamic",
+						"type": "string",
+						"in": "path",
+						"description": "Select the Table you want to get items from.",
+						"required": true,
+						"x-ms-summary": "Select Source Table",
+						"x-ms-url-encoding":"single",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					},
+					{
+						"in": "path",
+						"name": "itemId",
+						"required": true,
+						"description": "Numeric ID of the item to get",
+						"type": "integer",
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Item ID"
+					},
+					{
+						"in": "body",
+						"name": "Attachments",
+						"required": true,
+						"description": "Numeric ID of the item to get",
+						"x-ms-summary": "Attachments",
+						"schema": {
+							"type": "array",
+							"items": {
+								"type": "object",
+								"properties": {
+									"drive_id": {
+										"type": "string",
+										"x-ms-summary": "Drive ID"
+									},
+									"drive_item_id": {
+										"type": "string",
+										"x-ms-summary": "Drive Item ID"
+									}
+								}
+							}
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "XXX SOMETHING",
```

Comment:
@MattFranklin-Rapid Can you please update it with valid response description.

Reply:
Added missing template description to attachments response action